components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -89,6 +89,28 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
 
 An [environment]({{< relref "environments" >}}) can be used to map an organization's real-life workflow to what can be configured and managed when using Chef Infra. This mapping is accomplished by setting attributes and pinning cookbooks at the environment level.
 
+The Chef Infra Server UI let's you access the _Environment_ object in following ways:
+
+- All the _Environments_ of an Organization can be listed at one place.
+
+- You can create a new Environment using the button as shown below:
+
+{{< figure src="/images/automate/create-environment-button.png" alt="Create Environment Button">}}
+
+- In the above image, the search bar stating __Search environments__ can be used to search a specific environment from the list.
```

Comment:
Please use `**` to bold words. It is a convention that helps anyone reading the markdown understand your intent.